packages/code-du-travail-frontend/src/outils/common/StepList.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import styled from "styled-components";
+import { FlatList, theme, Wrapper } from "@socialgouv/react-ui";
+
+export const STEP_LIST_WIDTH = "28rem";
+
+export function StepList({ activeIndex = 0, items = [] }) {
+  return (
+    <StyledWrapper variant="dark">
+      <Title>
+        Étape<Plural>s</Plural>
```

Comment:
Plural is misleading, I would prefer <HideOnMobile />

Reply:
I guess you tried to add and html tag without the comment here :D Does `<S>`is ok for you ?

Reply:
I updateed the comment, I've forget to escape it  `<HideOnMobile />`